llm-complete-guide/constants.py

```diff
@@ -85,3 +85,5 @@
 # Vector store secrets
 SECRET_NAME = os.getenv("ZENML_PROJECT_SECRET_NAME", "llm-complete")
 SECRET_NAME_ELASTICSEARCH = "elasticsearch-zenml"
+
+DEFAULT_PROMPT = "Answer the question."
```

Comment:
?

Reply:
Prompt was not optional on most functions so I created a default one - this is one is obviously bad

Reply:
better now?